### ISSUE TITLE
Segmentation faut on android 9

### ISSUE THREAD

**Issue by andia89** [Genymobile/scrcpy#515]
I build the program (on linux) according to the instructions and everything works fine

However, when running `scrcpy` in terminal I get a segmenation fault :/

Here the error messages from `adb logtrace | grep " E "`

```
04-30 15:51:37.882 23311 23311 E libc    : Access denied finding property "ro.vendor.at_library"
04-30 15:51:37.948  1070  1996 E OMX-VENC: Enable/Disable allocate-native-handle allowed only in secure session
04-30 15:51:37.948  1070  1996 E OMXNodeInstance: setParameter(0xe80cd184:qcom.encoder.avc, OMX.google.android.index.allocateNativeHandle(0x7f00005d): Output:1 en=0) ERROR: UnsupportedSetting(0x80001019)
04-30 15:51:37.948  1070  1996 E OMX-VENC: set_parameter: metamode is valid for input port only
04-30 15:51:37.953  1070  1070 E OMX-VENC: venc_dev: Unsupported eColorFormat 0x7f000789
04-30 15:51:37.955  1070  1070 E OMX-VENC: Failed to update the profile_level
04-30 15:51:37.956  1070  1070 E OMX-VENC: ERROR: unsupported index 1870659589
04-30 15:51:37.956  1070  1070 E OMXNodeInstance: getConfig(0xe80cd184:qcom.encoder.avc, ConfigLatency(0x6f800005)) ERROR: UnsupportedIndex(0x8000101a)
04-30 15:51:37.969  1070  1996 E OMX-VENC: Does not handle dataspace request
04-30 15:51:37.969  1070  1996 E OMXNodeInstance: getConfig(0xe80cd184:qcom.encoder.avc, ??(0x7f000062)) ERROR: UnsupportedSetting(0x80001019)
04-30 15:51:37.979   708   977 E         : HpInterface: Wrong interface type.
04-30 15:51:37.996   708   708 E HWComposer: getHdrCapabilities failed for display -1: Invalid display
04-30 15:51:38.001   708   708 E HWComposer: getSupportedPerFrameMetadata failed for display -1: Invalid display
04-30 15:51:38.031  1070 23324 E OMX-VENC: venc_dev: id = 10035242, value = 4294967295, min = 0, max = 0
04-30 15:51:38.031  1070 23324 E OMX-VENC: venc_dev: id = 10035242, value = 4294967295, min = 0, max = 0
04-30 15:51:38.033  1070 23326 E OMXNodeInstance: setConfig(0xe80cd184:qcom.encoder.avc, ??(0x7f000062)) ERROR: UnsupportedSetting(0x80001019)
04-30 15:51:38.078  1070 23325 E OMX-VENC: venc_dev: async_venc_message_thread interrupted to be exited
```

**Comment by rom1v** [comment]
Which device is it?

The segfault occurs on the device, right? (not the client)

**Comment by Cris70** [comment]
Hi, I don't know if it is the same problem, but I have a segmentation fault with my Xiaomi 9 SE (with Android 9).

Here is what I get at the command line:
```
~> scrcpy
INFO: scrcpy 1.11 <https://github.com/Genymobile/scrcpy>
/usr/share/scrcpy/scrcpy-server: 1 file pushed. 1.7 MB/s (35294 bytes in 0.020s)
Segmentation fault
```

And here is what I get from the command `adb logcat | grep " E "` as soon as I launch the command above:
```
12-10 22:26:27.696  2479  2479 E crash_dump64: AM data write failed: Broken pipe
12-10 22:26:27.696  2479  2479 E crash_dump64: cannot open libmiuindbg.so: No such file or directory
12-10 22:26:27.698  1004  1004 E /system/bin/tombstoned: Tombstone written to: /data/tombstones/tombstone_08
12-10 22:26:32.038  1883  2433 E CloudServerSyncAdapterBase: getPhoneInfo: 
12-10 22:26:32.038  1883  2433 E CloudServerSyncAdapterBase: java.io.IOException
12-10 22:26:32.038  1883  2433 E CloudServerSyncAdapterBase:    at com.xiaomi.accountsdk.activate.ActivateSdkHelper$AmTaskSession.convertErrorCodeToException(ActivateSdkHelper.java:421)
12-10 22:26:32.038  1883  2433 E CloudServerSyncAdapterBase:    at com.xiaomi.accountsdk.activate.ActivateSdkHelper$AmTaskSession.access$100(ActivateSdkHelper.java:308)
12-10 22:26:32.038  1883  2433 E CloudServerSyncAdapterBase:    at com.xiaomi.accountsdk.activate.ActivateSdkHelper$AmTaskSession$2.onError(ActivateSdkHelper.java:332)
12-10 22:26:32.038  1883  2433 E CloudServerSyncAdapterBase:    at com.xiaomi.accountsdk.activate.IActivateServiceResponse$Stub.onTransact(IActivateServiceResponse.java:66)
12-10 22:26:32.038  1883  2433 E CloudServerSyncAdapterBase:    at android.os.Binder.execTransact(Binder.java:733)
12-10 22:26:32.078  1883  2433 E CloudServerSyncAdapterBase: getPhoneInfo: SIM not inserted or not activated for sim:1
12-10 22:26:32.078  1883  2433 E MiCloudSyncAdapterBase: onPerformSync server exception, reason: activated_fail
12-10 22:26:36.781  1883  2438 E CloudServerSyncAdapterBase: getPhoneInfo: 
12-10 22:26:36.781  1883  2438 E CloudServerSyncAdapterBase: java.io.IOException
12-10 22:26:36.781  1883  2438 E CloudServerSyncAdapterBase:    at com.xiaomi.accountsdk.activate.ActivateSdkHelper$AmTaskSession.convertErrorCodeToException(ActivateSdkHelper.java:421)
12-10 22:26:36.781  1883  2438 E CloudServerSyncAdapterBase:    at com.xiaomi.accountsdk.activate.ActivateSdkHelper$AmTaskSession.access$100(ActivateSdkHelper.java:308)
12-10 22:26:36.781  1883  2438 E CloudServerSyncAdapterBase:    at com.xiaomi.accountsdk.activate.ActivateSdkHelper$AmTaskSession$2.onError(ActivateSdkHelper.java:332)
12-10 22:26:36.781  1883  2438 E CloudServerSyncAdapterBase:    at com.xiaomi.accountsdk.activate.IActivateServiceResponse$Stub.onTransact(IActivateServiceResponse.java:66)
12-10 22:26:36.781  1883  2438 E CloudServerSyncAdapterBase:    at android.os.Binder.execTransact(Binder.java:733)
12-10 22:26:36.822  1883  2438 E CloudServerSyncAdapterBase: getPhoneInfo: SIM not inserted or not activated for sim:1
12-10 22:26:41.622 12732 12732 E GsaVoiceInteractionSrv: updateScreenStatus
12-10 22:26:41.623 12732 12732 E HotwordState: FASE: true, ACR: true, HR: true, ACbSS: false, ACbHE: false, SO:true, ARS: false
```

Thank you in advance for your support.
Cris

**Comment by rom1v** [comment]
@Cris70 Use the latest version (it's #944, fixed since v1.12).

**Comment by Cris70** [comment]
@rom1v thank you! I'll try it as soon as it is available in the repos of my distro.

**Comment by rom1v** [comment]
Probably fixed in v1.12. Feel free to reopen if you can reproduce on the latest version.
